update to python tutorial configuration file (#164)
microsoft/winget-dsc: samples/Templates/Introduction/Python 3.13/README.md

# Understanding WinGet Configuration Files

This folder contains a [Windows Package Manager](https://learn.microsoft.com/windows/package-manager/winget/) (WinGet) [Configuration File](https://learn.microsoft.com/windows/package-manager/configuration/) (_learn_python.winget_) that will work with the WinGet command line interface (`winget configure --file [path: learn_python.winget]`) or can be run directly by double-clicking on the file.

When run, the `learn_python.winget` file will install the following list of applications:

- Python 3.13
- Visual Studio Code
- Visual Studio Code extension for Python

> - Git (optional)

Reviewer: ```suggestion
```

If anything is already installed, it will skip that item. 

## How to use the WinGet Configuration File

### File Explorer

1. Download the `learn_python.winget` file to your computer.
2. Double-click the `learn_python.winget` file.

### Command line

1. Download the `learn_python.winget` file to your computer.
2. Open your Windows Start Menu, search and launch "_Windows Terminal_".

> 3. Type the following: `CD <C:\Users\User\Download>`

Reviewer: ```suggestion
3. Navigate to the path with the WinGet Configuration File: `CD <C:\Users\<Username>\Download>`
```

> 4. Type the following: `winget configure --file .\learn_python.winget`

Reviewer: ```suggestion
4. Run: `winget configure --file learn_python.winget`
```

> This folder contains a [Windows Package Manager](https://learn.microsoft.com/windows/package-manager/winget/) (WinGet) [Configuration File](https://learn.microsoft.com/windows/package-manager/configuration/) (_configuration.winget_) that will work with the WinGet command line interface (`winget configure --file [path: configuration.winget]`) or can be run directly by double-clicking on the file.

Reviewer 2: ```suggestion
This folder contains a [Windows Package Manager](https://learn.microsoft.com/windows/package-manager/winget/) (WinGet) [Configuration File](https://learn.microsoft.com/windows/package-manager/configuration/) (_learn_python.winget_) that will work with the WinGet command line interface (`winget configure --file [path: learn_python.winget]`) or can be run directly by double-clicking on the file.
```
